package.json

```diff
@@ -30,25 +30,37 @@
   },
   "homepage": "https://github.com/fastify/fastify-static",
   "dependencies": {
-    "fastify-plugin": "^1.6.0",
+    "fastify-plugin": "^2.0.0",
     "glob": "^7.1.4",
     "readable-stream": "^3.4.0",
     "send": "^0.16.0"
   },
   "devDependencies": {
     "@types/node": "^10.14.9",
+    "@typescript-eslint/eslint-plugin": "^2.29.0",
+    "@typescript-eslint/parser": "^2.29.0",
     "concat-stream": "^2.0.0",
     "coveralls": "^3.0.4",
     "eslint-plugin-typescript": "^0.14.0",
-    "fastify": "^2.5.0",
-    "fastify-compress": "^0.8.1",
+    "fastify": "^3.0.0-alpha.1",
+    "fastify-compress": "github:thomheymann/fastify-compress#v3-upgrade",
     "pre-commit": "^1.2.2",
     "proxyquire": "^2.1.0",
     "simple-get": "^3.0.3",
     "snazzy": "^8.0.0",
     "standard": "^12.0.0",
+    "standardx": "^5.0.0",
     "tap": "^12.7.0",
-    "typescript": "^3.5.2",
-    "typescript-eslint-parser": "^22.0.0"
+    "tsd": "^0.11.0",
+    "typescript": "^3.5.2"
+  },
+  "tsd": {
+    "directory": "test/types"
+  },
+  "eslintConfig": {
+    "rules": {
+      "no-unused-vars": "off",
+      "@typescript-eslint/no-unused-vars": "error"
```

Comment:
I removed the `typescript-eslint-parser` because it was deprecated in favour of `typescript-eslint`. But that errors for interface imports. Based on standardjs [docs](https://standardjs.com/#typescript) this config is the workaround.